### PR TITLE
CLI: Add --remove-chunks= option to remove non-editable fields

### LINKED ISSUE #281: Option to remove specific chunks
Follow up to #163 

Add option to remove specific chunks: `--remove-chunks=[NAME]`

### PR DESCRIPTION
**Pull request by g-maxime**
Fixes #281

### CODE REVIEW

File: Source/Riff/Riff_Base.h [hunk 1]
```diff
@@ -559,7 +560,7 @@ protected :
     //---------------------------------------------------------------------------
     //Read/Write
     virtual void    Read_Internal       ();
-    virtual void    Modify_Internal     ()                                      {}
+    virtual void    Modify_Internal     ();
```
Comment (JeromeMartinez) [inline Source/Riff/Riff_Base.h:563]: no need

File: Source/Riff/Riff_Handler.cpp [hunk 2]
```diff
@@ -1405,6 +1413,41 @@ bool Riff_Handler::Remove_Internal(const string &Field)
...
+    }
+
+    int32u CC4=Ztring().From_UTF8(Field).To_CC4();
+    if (!CC4)
+    {
+        Errors<<"Invalid chunk identifier: "<<Field<<endl;
+        return false;
+    }
+
+    if (CC4==Elements::WAVE_data || CC4==Elements::WAVE_ds64)
```
Comment (JeromeMartinez) [inline Source/Riff/Riff_Handler.cpp:1439]: Add "fmt " to the list.

**Review by JeromeMartinez — CHANGES_REQUESTED** [review]

File: Source/Riff/Riff_Base.h (hunk 1 above)
Comment (g-maxime) [reply Source/Riff/Riff_Base.h:563]: Definition is here: https://github.com/MediaArea/BWFMetaEdit/pull/283/files#diff-d451cf3c5401ca8dbca30e9caf315341c7ba564f909a93b642eb39e4c1e4c42aR650

**Review by JeromeMartinez — APPROVED** [review]

**Comment by g-maxime** [comment]
Snapshot: https://mediaarea.net/download/snapshots/binary/bwfmetaedit/20240915-2

Subchunks can now be removed by specifying the path, e.g. `--remove-chunks=adtl/labl` to remove all cue labels.

**Comment by MarcosSueiro** [comment]
Terrific, thank you!

Marcos Sueiro Bal
Archive Manager
New York Public Radio
646 829 4063
160 Varick St
New York NY 10013
USA

**Pull request merged**